src/cockpit/packages.py

```diff
@@ -223,11 +228,17 @@ def add_file(self, item: Path, checksums: List['hashlib._Hash']) -> None:
         # file then add it to the list of translations, by locale name.
         # Otherwise, add it to the normal files list (after stripping '.gz').
         po_match = Package.PO_JS_RE.fullmatch(rel)
+        po_manifest_match = Package.PO_MANIFEST_RE.fullmatch(rel)
```

Comment:
Why not do this as a single RE with the `.manifest` optional?  Then you could detect which of the two cases it is from the common `if:` block, after the match.

It might also be nice to call it `po-manifest` or so, to avoid any ambiguity about what follows `po.` (ie: the string `manifest`  can never be interpreted as a locale).